src/components/DopplerPlus/UpgradeSuggestionForm.js

```diff
@@ -0,0 +1,187 @@
+import React, { useState } from 'react';
+import { Helmet } from 'react-helmet';
+import HeaderSection from '../shared/HeaderSection/HeaderSection';
+import { InjectAppServices } from '../../services/pure-di';
+import { Field } from 'formik';
+import {
+  EmailFieldItem,
+  FieldGroup,
+  FieldItem,
+  FormWithCaptcha,
+  InputFieldItem,
+  PhoneFieldItem,
+  SubmitButton,
+} from '../form-helpers/form-helpers';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+const UpgradeSuggestionForm = ({ dependencies: { dopplerLegacyClient, appSessionRef } }) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
+  const fieldNames = {
+    email: 'email',
+    firstname: 'firstname',
+    lastname: 'lastname',
+    phone: 'phone',
+    range_time: 'range_time',
+    message: 'message',
+  };
+
+  const getFormInitialValues = () => {
+    const initialValues = Object.keys(fieldNames).reduce(
+      (accumulator, currentValue) => ({
+        ...accumulator,
+        [currentValue]: '',
+      }),
+      {},
+    );
+
+    initialValues[fieldNames.email] = appSessionRef.current.userData.user.email;
+
+    return initialValues;
+  };
+
+  const [formSubmitted, setFormSubmitted] = useState(false);
```

Comment:
We are trying to keep order like we said in slack:
- Hooks first (all useState, useEffect useIntl on top)
- Constants related to props
- Functions
- Then constants that have no relation to props outside the component (fieldNames here)